### DISCUSSION TITLE
Underworld Armament Set Items

### DISCUSSION THREAD

**Discussion by Akinraze** [flareteam/flare-game#875] [Q&A] [answered]
According to the Underworld Armament Set details, one should be able to equip 7 of this sets items to attain the set bonuses.
It is impossible to equip all seven items as several of the items are class specific.

Example. You cannot equip a Sceptre, Sword and Bow simultaneously
Is there something I am missing that we can do or is this as it seems?

**Comment by WithinAmnesia** [comment, marked as answer]
https://opengameart.org/forumtopic/underworld-armament-set#comment-94964

**Reply by Akinraze** [reply]
Interesting..... I guess I better start really doing runs to get them all.

**Comment by dorkster** [comment]
Also keep in mind the stat requirements for a full set. You'll need at least:

- 14 Physical
- 16 Mental
- 15 Offense
- 28 Defense
- One of Physical/Mental/Offense must be at least 30, depending on which of the three weapons you want to use

The set is pretty over powered, so actually finding all the pieces and getting the required stats is designed to be difficult/uncommon.

**Reply by Akinraze** [reply]
Almost level 17 now. I'm sure the point requirements for 18 are not easily attained.

**Comment by Akinraze** [comment]
If you are concerned, this map is highly repeatable in it's current design. I can get in 4 runs per magic find potion, very easily.
Runs: [https://drive.google.com/file/d/1jJUiFefJCuElD34C2SZtOTzW3swpkZsM/view?usp=sharing](url)
Yields: [https://drive.google.com/file/d/10AyHbaSpEzvsV1T6DzPHlXlLB_N2lhDR/view?usp=sharing](url)
